DISASTER_RECOVERY.md

```diff
@@ -0,0 +1,48 @@
+# Disaster recovery
```

Comment:
Can you move the file to the `docs` directory and write the name in lower case?

Reply:
Done